core/chains/evm/config/config_test.go

```diff
@@ -232,8 +232,8 @@ func TestChainScopedConfig_GasEstimator(t *testing.T) {
 	assert.Equal(t, assets.GWei(20), ge.PriceDefault())
 	assert.Equal(t, assets.GWei(500), ge.PriceMax())
 	assert.Equal(t, assets.GWei(1), ge.PriceMin())
-	assert.Equal(t, uint64(500000), ge.LimitDefault())
-	assert.Equal(t, uint64(500000), ge.LimitMax())
+	assert.Equal(t, uint64(8000000), ge.LimitDefault())
+	assert.Equal(t, uint64(8000000), ge.LimitMax())
```

Comment:
Go supports the `_` format too:
```suggestion
	assert.Equal(t, uint64(8_000_000), ge.LimitDefault())
	assert.Equal(t, uint64(8_000_000), ge.LimitMax())
```